lib/services/ui/chat/conversation_view_controller.dart

```diff
@@ -232,4 +233,27 @@ class ConversationViewController extends StatefulController with GetSingleTicker
     cm.setAllInactiveSync();
     Get.delete<ConversationViewController>(tag: tag);
   }
-}
\ No newline at end of file
+
+  Future<void> saveReplyToMessageState() async {
+    final prefs = await SharedPreferences.getInstance();
+    if (replyToMessage != null) {
+      await prefs.setString('replyToMessage_${chat.guid}', replyToMessage!.item1.guid!);
+      await prefs.setInt('replyToMessagePart_${chat.guid}', replyToMessage!.item2);
+    } else {
+      await prefs.remove('replyToMessage_${chat.guid}');
+      await prefs.remove('replyToMessagePart_${chat.guid}');
+    }
+  }
+
+  Future<void> loadReplyToMessageState() async {
+    final prefs = await SharedPreferences.getInstance();
```

Comment:
Same comment about the shared prefs instance here

Reply:
Dope Will do!